app/controllers/course/controller.rb

```diff
@@ -1,6 +1,8 @@
 class Course::Controller < ApplicationController
   load_and_authorize_resource :course
 
+  before_action :add_course_breadcrumb, only: [:show, :index, :edit]
```

Comment:
So this only affects one place? Which is when creating a new course?

except new would affect all new actions in all controllers.